scripts/gen-ceph-kustomize.sh

```diff
@@ -119,7 +122,12 @@ function get_pod_ip {
         echo "HOSTNETWORK must be set to true!"
         exit 1
     fi
-    MON_IP=$(oc get nodes -o 'jsonpath={.items[*].status.addresses[?(.type=="InternalIP")].address}')
+    NODES=$(oc get nodes -o 'jsonpath={.items[*].status.addresses[*].address}')
```

Comment:
@fmount I think we should only filter worker nodes

oc get nodes --selector='node-role.kubernetes.io/worker='

Reply:
+1, doing that in my next update.

Reply:
Done